src/Styleguide/Pages/Artwork/Sidebar/PartnerInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { Serif } from "@artsy/palette"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Flex } from "Styleguide/Elements/Flex"
+import { Location } from "../../../../Assets/Icons/Location"
+
+export interface PartnerInfoProps {
+  artwork?: {
```

Comment:
This part is also not cleat - maybe this component should take artwork  and partner is embedded in artwork? Not sure....

Reply:
Yup, that’s how I would model it, because that’s also how we’d do it with a Relay fragment container.